src/index.html

```diff
@@ -11,7 +11,159 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:3000/api" method="post">
+
+
+      <fieldset enabled class="forma">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname"
+            class="field"
+
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text" name="name"
+          class="field"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input type="number" name="number"
+           class="field"
+          min="1" max="100"
+          value="12"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="date"
+          class="field"
+          >
+
+        </label>
+
+
+        <label>
+            I accept the term of the agreement
+          <input type="checkbox" name="I accept the term of the agreement"
+           class="field"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+           E-mail:
+          <input type="email" name="email"
+           class="field"
+           required
+          placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password:
+          <input type="password" name="password"
+          class="field"
+          autocomplete="off"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset enabled class="forma">
+        <legend>An interesting fact about you!</legend>
+
+
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="answer"
+             class="field"
+            value="y"
+          >
+          Yes
+          <input type="radio" name="answer"
+            class="field"
+            value="n"
+          >
+          No
+        </label>
+        <label class="form-field">
```

Comment:
let's cover all inputs in label tags

https://user-images.githubusercontent.com/70220667/227763366-43a975a5-874f-4483-8dda-7c963f7289d6.mov